buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -1256,12 +1255,8 @@ private Path getExtractedDistributionDir() {
     }
 
     @Input
-    public Set<URI> getRemotePlugins() {
-        Set<URI> file = plugins.stream()
-            .map(Provider::get)
-            .filter(uri -> uri.getScheme().equalsIgnoreCase("file") == false)
-            .collect(Collectors.toSet());
-        return file;
+    public Set<File> getRemotePlugins() {
```

Comment:
I think this can be completely removed as we no longer allow installing remote plugins via URI. These files will otherwise be tracked via `getInstalledFiles()`.